docs/installation/index.html

```diff
@@ -102,130 +105,165 @@ <h3>Usage Example</h3>
 		Linux Raspberry Pi: <a href="https://github.com/duckdb/duckdb/releases/download/master-builds/libduckdb-linux-rpi.zip" target="_blank">https://github.com/duckdb/duckdb/releases/download/master-builds/libduckdb-linux-rpi.zip</a>
 	</div>
-	
+
 	<div class="master cplusplus binary win">
 		Win 32-bit: <a href="https://github.com/duckdb/duckdb/releases/download/master-builds/libduckdb-windows-i386.zip" target="_blank">https://github.com/duckdb/duckdb/releases/download/master-builds/libduckdb-windows-i386.zip</a><br/><br/>
 		Win 64-bit: <a href="https://github.com/duckdb/duckdb/releases/download/master-builds/libduckdb-windows-amd64.zip" target="_blank">https://github.com/duckdb/duckdb/releases/download/master-builds/libduckdb-windows-amd64.zip</a>
 	</div>
-	
+
 	<div class="master cli binary macos">
 		<a href="https://github.com/duckdb/duckdb/releases/download/master-builds/duckdb_cli-osx-amd64.zip" target="_blank">https://github.com/duckdb/duckdb/releases/download/master-builds/duckdb_cli-osx-amd64.zip</a>
 	</div>
-	
+
 	<div class="master cli binary linux">
 		Linux 64-bit: <a href="https://github.com/duckdb/duckdb/releases/download/master-builds/duckdb_cli-linux-amd64.zip" target="_blank">https://github.com/duckdb/duckdb/releases/download/master-builds/duckdb_cli-linux-amd64.zip</a><br/><br/>
 		Linux 32-bit: <a href="https://github.com/duckdb/duckdb/releases/download/master-builds/duckdb_cli-linux-i386.zip" target="_blank">https://github.com/duckdb/duckdb/releases/download/master-builds/duckdb_cli-linux-i386.zip</a><br/><br/>
 		Linux Raspberry Pi: <a href="https://github.com/duckdb/duckdb/releases/download/master-builds/duckdb_cli-linux-rpi.zip" target="_blank">https://github.com/duckdb/duckdb/releases/download/master-builds/duckdb_cli-linux-rpi.zip</a>
 	</div>
-	
+
 	<div class="master cli binary win">
 		Win 64-bit: <a href="https://github.com/duckdb/duckdb/releases/download/master-builds/duckdb_cli-windows-amd64.zip" target="_blank">https://github.com/duckdb/duckdb/releases/download/master-builds/duckdb_cli-windows-amd64.zip</a><br/><br/>
 		Win 32-bit: <a href="https://github.com/duckdb/duckdb/releases/download/master-builds/duckdb_cli-windows-i386.zip" target="_blank">https://github.com/duckdb/duckdb/releases/download/master-builds/duckdb_cli-windows-i386.zip</a>
 	</div>
-	
-	
-	
-	
-	
-	
+
+	<div class="master odbc binary macos">
+		<a href="https://github.com/duckdb/duckdb/releases/download/master-builds/libduckdb-osx-amd64.zip" target="_blank">https://github.com/duckdb/duckdb/releases/download/master-builds/libduckdb-osx-amd64.zip</a>
+	</div>
+
+	<div class="master odbc binary linux">
+		Linux 64-bit: <a href="https://github.com/duckdb/duckdb/releases/download/master-builds/libduckdb-linux-amd64.zip" target="_blank">https://github.com/duckdb/duckdb/releases/download/master-builds/libduckdb-linux-amd64.zip</a><br/><br/>
```

Comment:
These files don't appear to be there, perhaps it would be better to leave these empty if there is no possibility of downloading master binaries for the ODBC build?

Reply:
Removed the links from C++ and ODBC tabs.